frontend/testing/playwright/pages/DataModelPage.ts

```diff
@@ -72,17 +85,16 @@ export class DataModelPage extends BasePage {
     await this.page.getByRole('treeitem', { name }).focus();
   }
 
-  public async clickOnAddNodeToPropertyButton(): Promise<void> {
+  public async clickOnAddPropertyToObjectButton(propertyName: TextKey): Promise<void> {
     await this.page
-      .getByRole('button', { name: this.textMock('schema_editor.add_node_of_type') })
-      .last()
+      .getByRole('menuitem', {
+        name: this.textMock(propertyName),
```

Comment:
Then adjust like this: 
```suggestion
        name: this.textMock(`schema_editor.add_${property}`)),
```